src/modules/facets/renderSelectedFacets.js

```diff
@@ -34,15 +39,19 @@ export default function() {
         const l = r[j];
         const val = selectedRanges[l];
         val.forEach(rEl => {
-            selectedUi += this.options.facet.selectedFacetItemTemplate.bind(this)({
+            selectedUi += itemTemplate.bind(this)({
                 facetName: l,
                 facetType: "range"
             }, {
                 name: rEl.replace(/[^\w\s]/gi, ''),
                 dataId: rEl
-            })
+            },facet,selectedFacets)
         })
     }
-    this.selectedFacetWrapper.innerHTML = this.options.facet.selectedFacetTemplate(selectedUi, facet);
+    let selectedFacetTemp = selectedFacets.template.bind(this);
+    if(facet.selectedFacetTemplate)  {
```

Comment:
the first priority should be given to `selectedFacets.template`. If that is not present then use the deprecated `facet.selectedFacetTemplate` . The above logic does the opposite.

Reply:
we are getting the first options as new config only, i guess if we add this check `facet.selectedFacetsEl` to make sure that customer is using the old config.